src/main/java/uk/ac/sanger/sccp/stan/model/MeasurementValueType.java

```diff
@@ -6,5 +6,5 @@
 public enum MeasurementValueType {
     INT, // An integer
     TIME, // A duration in seconds
-    DECIMAL_2, // A positive or negative rational number with two decimal places
+    DECIMAL, // A positive or negative rational number with decimal places
```

Comment:
Just to clarify, is the measurement DECIMAL value no more restricted to 2 decimal places?

Reply:
No. The DECIMAL_2 value type was used for concentration, which is restricted to two decimal places. I've replaced that with DECIMAL, which is also used for the DV200 measurement, but that one is one decimal place, and I removed the specification here about how many decimal places. The validators still check elsewhere in the code still check that inputs are the correct format.